uniswap-factory-pool-monitoring/actions/example.ts

```diff
@@ -0,0 +1,47 @@
+const ethers = require('ethers');
+import { Network, Context, AlertEvent } from '@tenderly/actions';
+const axios = require('axios');
+
+// Do not change function name.
+const actionFn = async (context: Context, alertEvent: AlertEvent) => {
+
+	const provider = new ethers.providers.JsonRpcProvider(context.gateways.getGateway(Network.MAINNET));
+	const key = await context.secrets.get('ACCESS-KEY')
+	const networkId = "1";
+	const tagName = "child";
+
+	const resp = await provider.send("tenderly_traceTransaction", [`${alertEvent.hash}`]);
+	const poolCreatedLogs = resp.logs.filter((log: { name: string; }) => log.name === 'PoolCreated');
+	const poolInput = poolCreatedLogs[0].inputs.find((input: { name: string; }) => input.name === 'pool');
```

Comment:
The alert can hook up directly to the `PoolCreated` event, instead of monitoring every successful transaction. Then you can use `alertEvent.logs` and see if there is a `PoolCreated` event there.

Reply:
additional thing, what this code does is that it looks inside tx that triggered alert  for `PoolCreated` event and then extracts the `pool` value directly. Do you have other idea in mind, and is Node use here 'too much' ?

Reply:
Yes, let's go with the least amount of friction (the logs that are a part of the `alertEvent`).